Containerfile.redis

```diff
@@ -19,3 +19,6 @@ RUN make BUILD_TLS=yes && \
 
 # Expose the default Redis port
 EXPOSE 6379
+
+# Keep the container awake
+ENTRYPOINT [ "sleep", "infinity" ]
```

Comment:
how about spining up Redis with default config? 
`ENTRYPOINT [ "redis-server", "/etc/redis/redis.conf" ]`

Reply:
Sounds good if someone want to use our image alone.